src/arrayMethodSort.js

```diff
@@ -4,8 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+        }
+      }
+    }
+
+    return this;
```

Comment:
Returning 'this' is correct for chaining, but ensure that the sort method modifies the array in place without creating a copy.